app/frontend/src/components/NavigationBar.vue

```diff
@@ -20,15 +20,15 @@ export default {};
 <style scoped>
 .navigation-main {
   display: block;
-  position: fixed;
+  /* position: fixed; */
```

Comment:
Open the app in Chrome with dev tools open and large enough to require a vertical scroll in the content (secured page) and then scroll up and down.  You will see the navbar "float" in place and not stay "attached" to the header.  Applies to the footer too...  I think the footer was maybe picking up some stickiness from being class named "footer" and inheriting some styles - or - it was never sticky, as there was no content!  Anyway, the navbar and footer need to be sticky and have only the content area scroll.

Reply:
Saw it in action, was looking at deployed version, not Jeremy's.  Looks good!